uk.co.compucorp.civicrm.hremails/email-templates/src/layouts/leave-absences.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
+"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
+  <head>
+    <link rel="stylesheet" type="text/css" href="{{root}}css/app.css">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>{{subject}}</title>
+    <!-- <style> -->
+  </head>
+  <body>
+    <span class="preheader">{{description}}</span>
+    <table class="body">
+      <tr>
+        <td class="center" align="center" valign="top">
+          <center class="email">
+            <!-- header -->
+            <container class="email-heading">
+              <row class="collapse">
+                <columns small="12" large="6">
+                  <h1 class="email-title">CiviHR {{request-type}} Request</h1>
+                </columns>
+                <columns small="12" large="6">
+                  <p class="email-date small-text-left text-right">{$currentDateTime->format("D d F Y")}</p>
+                </columns>
+              </row>
+            </container>
+            <spacer></spacer>
+            <!-- body -->
+            <container>
+              {{> body}}
+            </container>
+          </center>
+        </td>
+      </tr>
+    </table>
+    <!-- prevent Gmail on iOS font size manipulation -->
+   <div style="display:none; white-space:nowrap; font:15px courier; line-height:0;"> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </div>
```

Comment:
This comes as part of the default layout in Foundation for emails, it's some hack to "prevent Gmail on iOS font size manipulation"